subsys/bluetooth/host/Kconfig

```diff
@@ -96,10 +96,10 @@ config BT_RX_STACK_SIZE
 	depends on BT_HCI_HOST || BT_RECV_IS_RX_THREAD
 	default 1024
 	default 2048 if BT_MESH
-	default 512 if CONFIG_BT_HCI_RAW
-	range 1024 65536
+	default 512 if BT_HCI_RAW
+	range 512 65536 if BT_HCI_RAW
 	range 1100 65536 if BT_MESH
-	range 512 65536 if CONFIG_BT_HCI_RAW
+	range 1024 65536
 	help
```

Comment:
Are you sure the generic "catch all" range should be at the end when the generic default is the first in the list? I think we reached the decision of putting the generic default by trial-and-error, so that should be correct, so I'm surprised if the range directive has different semantics. The s/CONFIG_BT_HCI_RAW/BT_HCI_RAW/ is a good catch either way.